sdk-lib/src/main/java/cash/z/ecc/android/sdk/fixture/BlockRangeFixture.kt

```diff
@@ -0,0 +1,20 @@
+package cash.z.ecc.android.sdk.fixture
+
+import cash.z.ecc.android.sdk.model.BlockHeight
+import cash.z.ecc.android.sdk.model.ZcashNetwork
+
+object BlockRangeFixture {
+
+    @Suppress("MagicNumber")
+    private val BLOCK_HEIGHT_LOWER_BOUND = BlockHeight.new(ZcashNetwork.Mainnet, 1880001L)
```

Comment:
Based on a conversation with Pacu, we may want to add some ranges where the blocks are more full.  He suggests roughly some blocks in this range 1,730,000 - 1,800,000.  (We can play around to find a section that make sense.)

Reply:
Thanks for poking into this, as it appears that I had a bug related to working with checkpoint files 😬. I chose the range 1,730,001 - 1,730,100, as it takes about twice the time more than the previous range 1,880,001 - 1,880,100.